common/production_method_groups/imperia_industry.txt

```diff
@@ -0,0 +1,18 @@
+﻿pmg_charcoal_kiln = {
+	texture = "gfx/interface/icons/generic_icons/mixed_icon_base.dds"
```

Comment:
needs proper icon

Reply:
This is actually correct; this is the icon that shows up when you have mixed PMs activated.